components/prism-core.js

```diff
@@ -524,15 +524,21 @@ if (script) {
 	_.filename = script.src;
 
 	if (!_.manual && !script.hasAttribute('data-manual')) {
+		_.highlightAutomaticallyCallback = function () {
```

Comment:
I'm not a fan of exposing this function.
I guess that it's so you can still do `document.removeEventListener('DOMContentLoaded', Prism.highlightAutomaticallyCallback)`  but I don't think that this will help anyone. It doesn't preserve backward-compatibility and if we break things, we might as well force people to use `Prism.manual`.

Reply:
Yes, I exposed it for that reason. I agree that it would be redundant with either `Prism.manual` or `Prism.highlightAll()` and could be removed.